docs/release-verification.md

```
@@ -0,0 +1,101 @@
+```
+
+## 1. Verify SLSA provenance for Python wheel
+
+```bash
+gh attestation verify faultray-<VERSION>-py3-none-any.whl \
+  --owner mattyopon
+```
+
+Expected output includes:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)